Microsoft.Toolkit.Uwp.Services/Services/MicrosoftGraph/MicrosoftGraphAuthenticationHelper.cs

```diff
@@ -11,38 +11,25 @@
 // ******************************************************************
 
 using System;
-using System.IO;
-using System.Net;
 using System.Net.Http;
-using System.Text;
 using System.Threading.Tasks;
-using Microsoft.Identity.Client;
 using Microsoft.IdentityModel.Clients.ActiveDirectory;
-using Newtonsoft.Json;
-using Windows.Security.Authentication.Web;
+using Microsoft.Toolkit.Uwp.Services.Core;
```

Comment:
unused reference

Reply:
these all seemed to be used from what i could tell.

Reply:
fixed